dsl_parser/url_resolver/abstract_url_resolver.py

```diff
@@ -0,0 +1,31 @@
+#########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+import abc
+
+
+class AbstractUrlResolver(object):
```

Comment:
Following a previous comment, perhaps this should not be `AbstractUrlResolver` but rather `AbstratImportResolver`